gatsby-config.js

```diff
@@ -34,6 +35,91 @@ module.exports = {
                 includeInDevelopment: false,
             },
         },
+        {
+            resolve: `gatsby-plugin-feed`,
+            options: {
+                query: `
+                {
+                    site {
+                    siteMetadata {
+                        title
+                        siteUrl
+                    }
+                    }
+                }
+                `,
+                feeds: [
+                    {
+                        serialize: ({ query: { site, allSitePage } }) => {
```

Comment:
Any thoughts on pulling some stuff out of here for testing/readability? I had to revert the plugins import change since we apparently only have access to the prod config in production. We might want to put these in a shared place.

Reply:
that makes sense, I still want to revisit this a bit further as well